Lets Do This/Views/Campaign/LDTCampaignListCampaignCell.m

```diff
@@ -90,13 +96,29 @@ -(void)setExpanded:(BOOL)expanded {
 		self.titleLabelTopLayoutConstraint.constant = CGRectGetHeight(self.imageView.bounds)-CGRectGetHeight(self.titleLabel.bounds)-10; // -10 for padding
 		self.imageViewTop.constant = kCampaignImageViewConstantExpanded;
 		self.imageViewBottom.constant = kCampaignImageViewConstantExpanded;
-		
+        self.imageViewHeight.constant = 242;
+        self.actionButtonHeight.constant = 50;
+        self.actionButtonTop.constant = 16;
+        self.actionButtonBottom.constant = 16;
+		[self.actionButton enable:YES];
+        self.actionButton.hidden = NO;
+        self.taglineTop.constant = 16;
+        self.taglineLabel.hidden = NO;
+
 		[self layoutIfNeeded];
 	}
 	else {
 		self.imageViewTop.constant = kCampaignImageViewConstantCollapsed;
 		self.imageViewBottom.constant = kCampaignImageViewConstantCollapsed;
+        // Desired height is 150, but need 25 with the imageView constant.
+        self.imageViewHeight.constant = 175;
 		self.titleLabelTopLayoutConstraint.constant = self.collapsedTitleLabelTopLayoutConstraintConstant;
+        self.actionButton.hidden = YES;
+        self.actionButtonHeight.constant = 0;
+        self.actionButtonTop.constant = 0;
+        self.actionButtonBottom.constant = 0;
+        self.taglineTop.constant = 0;
```

Comment:
We're also hiding the button within the [`LDTCampaignDetailCampaignCell`](https://github.com/DoSomething/LetsDoThis-iOS/blob/5a598291492a58ac832cb344b31ca8dec61cf66e/Lets%20Do%20This/Views/Campaign/LDTCampaignDetailCampaignCell.m#L125).
This [answer on SO to use `clipsToSubviews`](http://stackoverflow.com/a/22463949/1470725) doesn't totally make sense, as it's a property on a `UIView` not a `NSLayoutConstraint`.